src/commands/check_workspace/mod.rs

```diff
@@ -1111,7 +1111,7 @@ pub async fn check_workspace(
     }
     for package_key in package_keys.clone() {
         if let Some(package) = packages.get_mut(&package_key) {
-            if package.changed {
+            if package.changed || package.dependencies_changed {
```

Comment:
I'm happy to add this, but I was asked to removed it end of last year

Reply:
I've documented the change and reasoning for the next time we forget and try to change it.